grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -657,11 +659,57 @@ public GrpcServiceBuilder enableHttpJsonTranscoding(boolean enableHttpJsonTransc
         return this;
     }
 
+    /**
+     * Sets whether the service handles HTTP/JSON requests using the gRPC wire protocol.
+     * Provide {@link HttpJsonTranscodingOptions} to customize HttpJsonTranscoding.
+     *
+     * <p>Example:
+     * <pre>{@code
+     * HttpJsonTranscodingOptions options =
+     *   HttpJsonTranscodingOptions.builder()
+     *                             .useCamelCaseQueryParams(true)
```

Comment:
We also need to update the Javadoc with the current API.

Reply:
Thanks for pointing this out 😅  Almost missed this